lib/menu.rb

```diff
@@ -0,0 +1,11 @@
+class Menu
+  The_menu = {
+    "Pad_thai" => 8,
+    "Red_curry" => 9,
+    "Green_curry" => 10
+    }
+
+  def show_menu
+    The_menu
+  end
```

Comment:
If  the only thing show_menu does is return The_menu, The_menu should be accessible by referring to the constant without having to write/call a method.